modules/keypad/commerce_pos_keypad.module

```diff
@@ -84,11 +85,15 @@ function commerce_pos_keypad_pre_render($element) {
  *   The altered element.
  */
 function commerce_pos_keypad_pre_render_cash_input(array $element) {
+  // Get all the denominations that the user has configured for.
+  $denominations = commerce_pos_currency_denomination_get_denominations(variable_get('commerce_pos_default_currency_denomination_types', array('USD')));
```

Comment:
Since you are calling a function in the commerce_pos_currency_denomination module from the commerce_pos_keypad module, you should add the former as a dependency of the latter in the .info file.